playground/AzureContainerApps/AzureContainerApps.AppHost/Program.cs

```diff
@@ -3,8 +3,17 @@
 
 #pragma warning disable ASPIREACADOMAINS001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
 
+using Azure.Provisioning.AppContainers;
+
 var builder = DistributedApplication.CreateBuilder(args);
 
+builder.AddContainerAppEnvironment("infra")
+       .ConfigureInfrastructure(infra =>
+       {
+           var env = infra.GetProvisionableResources().OfType<ContainerAppManagedEnvironment>().Single();
+           env.WorkloadProfiles.Clear();
+       });
```

Comment:
Thinking about this and the intersection of publishers. We will have a publisher for ACA-based deployments. I wonder if the `AddAzureContainerAppsPublisher(...)` we should have a callback in there where you can get at this rather than adding it as a resource?